Dockerfile

```diff
@@ -6,12 +6,14 @@ ENV TZ=America/New_York
 
 # Install system deps
 RUN apt-get update && \
-    apt-get install ffmpeg -y 
+    apt-get install --no-install-recommends ffmpeg=7:4.1.4-1~deb10u1 -y && \
```

Comment:
Why are we installing ffmpeg? We're not relying on it for anything at the moment, as far as I know.

Reply:
I wasn't sure if we'd need it for the background jobs stuff, its a holdover from the ss-shrine repo

Reply:
I agree. I'll remove that. The docker optimizations are ever-evolving right now 🚀